src/Monolog/Processor/PsrLogMessageProcessor.php

```diff
@@ -51,7 +51,7 @@ public function __invoke(array $record): array
             } elseif (is_object($val)) {
                 $replacements['{'.$key.'}'] = '[object '.get_class($val).']';
             } else {
-                $replacements['{'.$key.'}'] = '['.gettype($val).']';
+                $replacements['{'.$key.'}'] = 'array'.@json_encode($val);
```

Comment:
Feel free to add an elseif is_array() above and do this, but this last one should remain the same IMO